src/scss/alert/_mixins.scss

```diff
@@ -182,10 +172,10 @@
 			}
 
 			.#{$class}__button {
-				@include oMessageAlertButton(_oMessageGetMapValue($theme, button-background), _oMessageGetMapValue($theme, button-text));
+				@include oMessageAlertButtonTheme(_oMessageGetMapValue($theme, button-background), _oMessageGetMapValue($theme, button-text));
 			}
 
-			.#{$class}__content--header:before {
+			.#{$class}__content:before {
 				// hack mchackerson:
```

Comment:
about this ugli code that I wrote, James gave me some new icons, so I will upload them and then we can revisit this 🙈

Reply:
👍